buildstockbatch/gcp/gcp.py

```diff
@@ -87,15 +92,43 @@ def __init__(self, project_filename):
 
     @staticmethod
     def validate_gcp_args(project_file):
-        # TODO: validate GCP section of project definition, like region, machine type, etc
-        pass
+        cfg = get_project_configuration(project_file)
+        assert 'gcp' in cfg, 'Project config must contain a "gcp" section'
+        gcp_project = cfg['gcp']['project']
```

Comment:
rather than refetch all these values from cfg, can/should we validate the properties of GcpBatch (could make this non-static, or pass GcpBatch into this)?